src/main/webapp/about.jsp

```diff
@@ -49,8 +49,10 @@ User loggedInUser = UserStore.getInstance().getUser(user);
 
       <h1>About the CodeU Chat App</h1>
       <p>
-        This is a Chat App created by Spring 2018 CodeU Team 31!
```

Comment:
Any reason for the new whitespace? This won't add line breaks or anything; did you mean to wrap these in `<p>` tags?

Reply:
👍 got rid of the extra whitespace!

Reply:
Looks like it's still picking up a diff, with an extra line on 54 (and maybe line endings with spaces etc)

This is mostly a nit, but something I think is reasonable usually, since unnecessary diffs clutter the intent of the change and make me need to question what the scope of the PR is

Reply:
Should I move the changes to about.jsp and index.jsp to a different PR?